libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -299,6 +305,9 @@ public void hideInstructionList() {
       updateLandscapeConstraintsTo(R.layout.instruction_layout);
     }
     instructionListLayout.setVisibility(GONE);
+    if (instructionListListener != null) {
```

Comment:
We can remove duplicate code here by creating a function that takes a boolean (duplicate code: `L320-322`)